docs/src/pages/TextViewPage/TextViewPage.js

```diff
@@ -40,7 +39,6 @@ const TextViewPage = () => (
     title="Text views"
     blurb={blurb}
     iosSubpage={<IOS wrapped />}
```

Comment:
Shouldn't we move this to `components/text-input?platform=ios` too, to be consistent?

Reply:
Sure we can do I wasn't too sure here just due to this being a full view but if it makes sense then I can move it 😄

Reply:
Then we can kill off the textview page entirely

Reply:
Yeah it definitely makes sense - it's essentially a different type of text input

Reply:
Ok sounds good!